src/dvclive/report.py

```diff
@@ -131,11 +150,20 @@ def make_report(live: "Live"):
     if live._report_mode == "html":
         render_html(renderers, live.report_file, refresh_seconds=5)
     elif live._report_mode == "notebook":
-        from IPython.display import IFrame
-
-        render_html(renderers, live.report_file)
+        from IPython.display import HTML, IFrame
+
+        render_html(
+            renderers,
+            live.report_file,
+            # Use custom template to limit the size of the display
+            custom_template=COLAB_HTML if inside_colab() else None,
+        )
         if live._report_notebook is not None:
-            live._report_notebook.update(IFrame(live.report_file, "100%", 700))
+            if inside_colab():
+                new_report = HTML(live.report_file)
```

Comment:
For some reason, this works on Colab (displays the full report) but it doesn't display the plots in my local jupyter so I kept IFrame for that case